.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 10.x
+          node-version: 14.17.2
```

Comment:
What made this required? The `@testing-library` packages?

Reply:
Yep, then I updated it to last LTS version.